indico/modules/auth/controllers.py

```diff
@@ -204,6 +220,12 @@ class RHRegister(RH):
     - creation of a new user based on information from an identity provider
     """
 
+    def _checkProtection(self):
+        RH._checkProtection(self)
```

Comment:
RH._checkProtection is always empty, no need to call it. also, we never link to that page if account creation is not active so i'd raise either `NotFound` or `Forbidden` (both can be imported from `werkzeug.exceptions`) instead of the custom `AccessError`.